interfaces/policy/helpers.go

```diff
@@ -0,0 +1,73 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package policy
+
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/asserts"
+	"github.com/snapcore/snapd/snap"
+)
+
+// check helpers
+
+func checkSnapType(snapType snap.Type, types []string) error {
+	if len(types) == 0 {
+		return nil
+	}
+	s := string(snapType)
+	if s == "os" { // we use "core" in the assertions
+		s = "core"
+	}
```

Comment:
Hmmm, aren't we supposed to be moving to 'type: core' instead of 'type: os'? Since only the store reviewers are going to be working with this, perhaps it makes sense to forego this compatibility code?